test/Tests/src/Data/Vector_Spec.enso

```diff
@@ -449,6 +449,7 @@ type_spec name alter = Test.group name <|
         vec.slice 0 100 . should_equal [1, 2, 3, 4, 5, 6]
         Meta.is_same_object vec (vec.slice 0 100) . should_be_true
         Meta.get_qualified_type_name (vec.slice 1 1) . should_equal (Meta.meta Vector . qualified_name)
+        Meta.get_simple_type_name (vec.slice 1 1) . should_equal (Meta.meta Vector . qualified_name . take (Text_Sub_Range.After_Last '.'))
```

Comment:
I know I added the line above, but I'm actually curious - what is the simple name here? Is it just `Vector`?
Maybe we could
```suggestion
        Meta.get_simple_type_name (vec.slice 1 1) . should_equal "Vector"
```
to make this simpler?

Reply:
Do you know how much time it took me to find out the `Text_Sub_Range.After_Last` ;-?

Reply:
(partly joking, but maybe not) I guess it may be an argument for writing our unit tests in the GUI 😁 there I hope the discoverability of `take Last` should be better with our searcher :)